sequence-diagrams/host-provisioning-xPU-VF.md

```diff
@@ -47,51 +50,44 @@ as a Kubernetes deployment.
   - Configures the interface with the allocated IP address
   - Moves the VF/SF from the Host network namespace to the Pod Network namespace
    (vice versa).
-- **Broker**
-  - Syncs OPI CRDs between the Tenant and Infra clusters.
-  - The Broker is a singleton component that is deployed on a cluster whose Kubernetes API must be accessible by all of the participating clusters.
-  - The Broker cluster may be one of the participating clusters or a standalone cluster without the other components deployed. The Agent components deployed in each participating cluster are configured with the information to securely connect to the Broker cluster’s API.
 
-The following high level diagram provides an overview of the entities working
-together to provision a primary network interface for a pod.
+### Required Resources
 
-![primary-int-provisioning-highlevel](./images/primary-int-provisioning-highlevel.png)
+TODO (WIP)
```

Comment:
This is a non exhaustive list of resources that need to be shared between a tenant and infra cluster